script.js

```diff
@@ -0,0 +1,75 @@
+function deliverablesOne(q1) {
```

Comment:
Interesting approach, I like it. 

Something to ponder on.... instead of deliverableOne, dilverableTwo, etc.
Are you able to abstract the function into deliverable(product, 1);   deliverable(product, 2)